src/main/scala/com/databricks/labs/overwatch/pipeline/Schema.scala

```diff
@@ -258,11 +284,87 @@ object Schema extends SparkSessionWrapper {
     common("filenameGroup")
   ))
 
-  // Minimum required Schedule Schema
-  val minimumScheduleSchema: StructType = StructType(Seq(
-    StructField("pause_status", StringType, true),
-    StructField("quartz_cron_expression", StringType, true),
-    StructField("timezone_id", StringType, true)
+  val minimumLibrariesSchema: ArrayType = ArrayType(
+    StructType(Seq(
+      StructField("jar",StringType, nullable = true),
+      StructField("maven",
+        StructType(
+          Seq(StructField("coordinates",StringType, nullable = true)
+          )),nullable = true),
+      StructField("pypi",StructType(Seq(
+        StructField("package",StringType, nullable = true)
+      )), nullable = true),
+      StructField("whl",StringType, nullable = true)
+    )), containsNull = true
+  )
+
+  val minimumManualOverrideParamsSchema: StructType = StructType(Seq(
+    StructField("jar_params",ArrayType(StringType, containsNull = true), nullable = true),
+    StructField("python_params",ArrayType(StringType, containsNull = true), nullable = true),
+    StructField("spark_submit_params",ArrayType(StringType, containsNull = true), nullable = true),
+    StructField("notebook_params",ArrayType(StringType, containsNull = true), nullable = true)
```

Comment:
also add `python_named_params`, `pipeline_params`, `sql_params` and `dbt_commands`: https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunNow

You

Reply:
will add python_named_params but we're not fully support DLT yet and DLT and SQL tasks are still not really GA, I'm planning to give them some time to harden before I add them.

Reply:
also, jobs team is likely moving this to a new tag in the audit logs shortly, I'm working with them on the PR. Will likely change the way this works.